Error executing job with overrides: []
Traceback (most recent call last):
  File "/home/thich1998/Workspace/JaxMAPPO/baselines/MAPPO/sr_mappo_ff_overcooked.py", line 507, in main
    out = train_jit(rng)
          ^^^^^^^^^^^^^^
jaxlib.xla_extension.XlaRuntimeError: INTERNAL: Failed to execute XLA Runtime executable: run time error: custom call 'xla.gpu.custom_call' failed: CpuCallback error: KeyboardInterrupt: <EMPTY MESSAGE>
At:
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/interpreters/mlir.py(2266): _wrapped_callback
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/interpreters/pxla.py(1144): __call__
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/profiler.py(314): wrapper
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/pjit.py(1151): _pjit_call_impl_python
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/pjit.py(1195): call_impl_cache_miss
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/pjit.py(1211): _pjit_call_impl
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/core.py(869): process_primitive
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/core.py(389): bind_with_trace
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/core.py(2657): bind
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/pjit.py(166): _python_pjit_helper
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/pjit.py(255): cache_miss
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/jax/_src/traceback_util.py(177): reraise_with_filtered_traceback
  /home/thich1998/Workspace/JaxMAPPO/baselines/MAPPO/sr_mappo_ff_overcooked.py(507): main
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/core/utils.py(186): run_job
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/_internal/hydra.py(119): run
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/_internal/utils.py(458): <lambda>
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/_internal/utils.py(220): run_and_report
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/_internal/utils.py(457): _run_app
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/_internal/utils.py(394): _run_hydra
  /home/thich1998/miniconda3/envs/r2i_mappo/lib/python3.12/site-packages/hydra/main.py(94): decorated_main
  /home/thich1998/Workspace/JaxMAPPO/baselines/MAPPO/sr_mappo_ff_overcooked.py(521): <module>
; current tracing scope: custom-call.138; current profiling annotation: XlaModule:#hlo_module=jit_train,program_id=14#.
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.